pdcupdater/utils.py

```diff
@@ -92,6 +92,14 @@ def ensure_release_exists(pdc, release_id, release):
         release_payload.update(dict(
             active=True,
         ))
+
+        # XXX - Both release_type and type should have the same value, but PDC
+        # doesn't expect that the 'type' value should be present.  If it is
+        # present, we fail with a 400 error.  Here, we delete it until the
+        # pungi/PDC authors tell us otherwise.
+        if 'release_type' in release_payload and 'type' in release_payload:
+            del release_payload['type']
```

Comment:
OK, thinking a little more clearly now.

I think we just need to update our PDC instance in Fedora first, and then sort out if this is still a problem.  We should almost certainly drop our `release_type` field and use just the `type` field instead.